semgrep/semgrep/commands/scan.py

```diff
@@ -766,11 +764,9 @@ def scan(
     possibly_notify_user()
 
     # change cwd if using docker
-    try:
+    if not targets:
```

Comment:
is adjust_for_docker guaranteed to succeed if targets is not null?

Reply:
no; but the try-except here was just re-raising the exception anyway! i.e. we always wanted to crash if that function fails